runsc/boot/controller.go

```diff
@@ -788,6 +795,9 @@ type SignalArgs struct {
 // process group.
 func (cm *containerManager) Signal(args *SignalArgs, _ *struct{}) error {
 	log.Debugf("containerManager.Signal: cid: %s, PID: %d, signal: %d, mode: %v", args.CID, args.PID, args.Signo, args.Mode)
+	if cm.inCheckpointing.Load() != 0 {
+		return nil
```

Comment:
This doesn't look right either. We can't return success if the signal hasn't been sent....

Reply:
Thank you for your comments. Yes, I agree that returning success is not right. Currently, I do not have a solid design for this scenario, so I will close this PR and create an issue for further discussion.

Reply:
> This doesn't look right either. We can't return success if the signal hasn't been sent....

Thank you for your comments. Yes, I agree that returning success is not right. Currently, I do not have a solid design for this scenario, so I will close this PR and create an issue for further discussion.